### PR TITLE
LPS-75482 Undo changes not used

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
LGTM!

File: dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field_feedback_support.js [hunk 1]
```diff
@@ -141,10 +141,10 @@ AUI.add(
 			_showFeedback: function(icon) {
 				var instance = this;
 
-				instance.hideFeedback();
-
 				var container = instance.get('container');
 
+				instance.hideFeedback();
```
Comment (pedroqueiroz94) [inline dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field_feedback_support.js]: hey @alinedoleron , is this change really necessary?

**Comment by pedroqueiroz94** [comment]
@clovisdasilvaneto @interaminense can you guys review this too? Thanks!

**Comment by brunobasto** [comment]
LGTM
